packages/file_selector/file_selector_platform_interface/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.2.0
 
+* Makes XTypeGroup constant and changes the initialization of it.
```

Comment:
The second part of this is confusing since it's only referring to tests. How about just

```
* Makes `XTypeGroup`'s constructor constant.
```